docs/examples/guides/latest/auto-backup/database/backupconfiguration_template.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     storageSecretName: gcs-secret
   # ============== Template for BackupConfiguration =================
   task:
-    name: pg-backup-11.2
+    name: pg-backup-${TARGET_APP_VERSION}
```

Comment:
I think we should rename these tasks as `postgres-backup-${TARGET_APP_VERSION}`

Reply:
Fixed